draft-ietf-lamps-kyber-certificates.md

```diff
@@ -373,7 +376,9 @@ derived using `ML-KEM.KeyGen_internal(d,z)` (algorithm 16) using
 the first 32 octets as *d* and the remaining 32 octets as *z*.
 
 * The `expandedKey` format contains the expanded private key
-that was derived from the seed.
+that was derived from the seed. If the seed is not exported, both the
+expanded private key and public key can be derived using
+`ML-KEM.KeyGen()` (algorithm 16).
```

Comment:
This again makes little sense. Whether the seed is exported or not is irrelevant.

Reply:
If the seed is not exported, then ML-KEM.KeyGen() is a valid way of getting the expanded key for the `expandedKey` format.  But I would be okay if this change was removed.